### DISCUSSION TITLE
Toggle display order when >2 displays (for Next/Previous Display and Right/Left Half)

### DISCUSSION THREAD

**Discussion by jdanbrown** [rxhanson/Rectangle#365] [Ideas]
👋 Hello! It'd be helpful to add a toggle to reverse the order of displays in the "Next Display" / "Previous Display" operations.

I have three displays arranged like this, and "Next Display" traverses them counter-clockwise, whereas I'd like "Next Display" to instead mean clockwise.

![image](https://user-images.githubusercontent.com/627486/106184891-62a99380-615f-11eb-86ad-21472a34fa48.png)

This is easy enough to work around by switching my Next/Previous Display keys, but this doesn't suffice to also make repeated uses of "Right Half" / "Left Half" traverse the displays in the same way. I instead end up with Next/Previous cycling clockwise and Right/Left cycling counterclockwise, which is confusing.

Love the app! Thanks for all the great work!

**Comment by rxhanson** [comment]
Thanks for reporting. I think this is actually a bug related to the algorithm for determining which display is next or previous, with an alignment that involves vertically stacked displays. Here's the bug that's open for it: https://github.com/rxhanson/Rectangle/issues/192

I'll make a note to look at this more when I start working on the next release.
